activemq-unit-tests/src/test/java/org/apache/activemq/bugs/AMQ6131Test.java

```diff
@@ -58,6 +58,8 @@ public class AMQ6131Test {
 
     protected BrokerService broker;
     protected URI brokerConnectURI;
+    private final static WildcardFileFilter LOG_FILE_FILTER = WildcardFileFilter.builder().setWildcards("*.log").get();
```

Comment:
nit: private final static variable should be the first block of the class and the name should be all cap like WILDCARD_FILE_FILTER